Tests/Readme.md

```diff
@@ -15,11 +15,29 @@ For Xamarin.Mac, we're using a standalone runner that has no functionality other
 For Unity, we're building `Realm.Tests` as a .NET Standard 2.0 application that is referencing Unity's custom NUnit build and we're executing
 it either in the Editor or as a standalone player.
 
-## Test Explorer
-
-If you're using Visual Studio for Windows and running the tests with the test explorer, it might be a good idea to manually modify `Realm.Tests.csproj`
-and set `<LocalDev>true</LocalDev>`. What this will do is exclude all target frameworks except for `net461`. This dramatically speeds up VS builds as
-it forces it to only build for one platform.
+## Local development
+
+To facilitate local development setups, the following files are ignored, but it might be a good idea to create them manually and add them to your local clone:
+- `App.config`: this is a file that holds the config for your local Baas instance. Update `*your-local-ip*` with the local IP address of your docker image (note that `localhost` will not work):
```

Comment:
Would it be helpful to clarify where these two files have to be generated or is it clear from the fact that it's the Readme file within the Test folder? It was obvious to me since I've seen the .gitignore within the PR but might not be without that information.